lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -769,6 +781,35 @@ def test_following(self):
             "per_page": ["11"],
         })
 
+    @ddt.data("unanswered", "unread")
+    def test_view_query(self, query):
+        self.register_get_threads_response([], page=1, num_pages=1)
+        result = get_thread_list(
+            self.request,
+            self.course.id,
+            page=1,
+            page_size=11,
+            view=query,
+        )
+        self.assertEqual(
+            result,
```

Comment:
In the assertion below, we want to check that the view parameter is queried such that the comment_client_service (CCS) can use it. 

https://github.com/edx/edx-platform/pull/8571/files#diff-5a24451793a8ae129f1f78554dfef18eR810

If view=unanswered, then the query to the CCS should be unanswered=true.